alcs-frontend/src/app/features/application/documents/documents.component.html

```diff
@@ -25,14 +25,16 @@ <h3>Documents</h3>
   <ng-container matColumnDef="visibilityFlags">
     <th mat-header-cell *matHeaderCellDef mat-sort-header sortActionDescription="Sort by visibility">Visibility</th>
     <td mat-cell *matCellDef="let element">
-      <span *ngIf="element.visibilityFlags.includes('A')" matTooltip="Applicant">A, </span>
-      <span *ngIf="element.visibilityFlags.includes('C')" matTooltip="Commissioner"
-        >C<span *ngIf="!hasBeenSetForDiscussion">*</span>,
-      </span>
-      <span *ngIf="element.visibilityFlags.includes('G')" matTooltip="L/FNG">G, </span>
-      <span *ngIf="element.visibilityFlags.includes('P')" matTooltip="Public"
-        >P<span *ngIf="!hasBeenReceived">*</span></span
-      >
+      <span *ngIf="element.visibilityFlags.includes('A')" matTooltip="Applicant">A</span>
+      <ng-container *ngIf="element.visibilityFlags.includes('C')">
+        <span matTooltip="Commissioner">, C<span *ngIf="!hasBeenSetForDiscussion">*</span></span>
```

Comment:
What if `visibilityFlags` includes `'C'`, but not `'A'`?

Reply:
will check this case